src/design-tokens.yaml

```diff
@@ -79,7 +87,16 @@ background-color:
   danger: *color-red-30
   danger-dark: *color-red-60
   page-default: *color-white
-
+  
+container-color:
```

Comment:
The section above is called "background-color". Are these also background colors?

Reply:
I found a docs PR that answers my question, above. However, looks like John L. had a question about the name, too: https://github.com/blackbaud/skyux2-docs/pull/610#pullrequestreview-275210943

Reply:
Todd and I discussed the token scheme yesterday and I think we've got a better direction for naming and organizing these. Stay tuned.